### PR TITLE
Feature/Add TheOldSchool Provider

### PR DESCRIPTION
**Pull request by IamMika23**
Add TheOldSchool a french torrent provider

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/torrent/json/theoldschool.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        self.api_key = None
+
+        # URLs
+        self.url = 'https://theoldschool.cc'
+        self.urls = {
+            'search': urljoin(self.url, 'api/torrents/filter'),
+        }
+
+        # Proper Strings
+        self.proper_strings = ['{{PROPER|REPACK|REAL|RERIP}}']
```
Comment (p0psicles) [inline medusa/providers/torrent/json/theoldschool.py:38]: Not sure why you have the brackets in here. Like the. {{

**Review by p0psicles — DISMISSED** [review]

**Comment by p0psicles** [comment]
Overal nice pr. But you might want to fix the backend build

**Comment by medariox** [comment]
@IamMika23 
If you rebase the build tests should pass now.

**Review by medariox — APPROVED** [review]

**Comment by medariox** [comment]
LGTM, thanks!

**Pull request merged**